templates/Reviews.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
```

Comment:
Will reviews appear in separate page or in book page?

Reply:
erviews appear in book page while merging will make the changes needed

Reply:
you can change now, by the end of this review code should be ready to merge

Reply:
I have changed the name of the file with respect to the book page author.